region/brinstar/green/Spore Spawn Room.json

```diff
@@ -99,6 +99,41 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "Plasma SBA Kill from Above (Patient)",
+      "requires": [
+        {"notable": "Plasma SBA Kill from Above"},
+        "Plasma",
+        "canSpecialBeamAttack",
+        {"ammo": {"type": "PowerBomb", "count": 1}},
+        "canBePatient"
+      ],
+      "setsFlags": ["f_DefeatedSporeSpawn"],
+      "note": [
+        "Use the Plasma Special Beam Attack to kill Spore Spawn from above.",
+        "Spore Spawn will move out of range after the first hit and it will take 9 cycles to come back into range.",
+        "Stand on the left side of the lowest part of the upper area, facing left for the first hit.",
+        "9 cycles later, shoot again from the same spot while crouching to kill Spore Spawn."
```

Comment:
Curious if you checked to see if these work on both vanilla with the broken camera, and on Map Rando with the camera fixed. Hopefully both are the same

Reply:
I conveniently have a save file near sporespawns top.
The slow method works.
The easy xray method doesn't work.  But I had a trickier way to use microwave which does work: https://videos.maprando.com/video/4262  Only you can't see what is happening too well so its harder than the video shows.